### ISSUE TITLE
No indication of what opponent are doing when you have finished your deck

### ISSUE THREAD

**Issue by Zomis** [Cardshifter/HTML-Client#87]
When you have finished your deck and clicked "Start game", there is no indication that you are actually waiting for opponent to build his/her deck.

Additionally, there's no guarantee that the opponent actually *will* finish their deck, they might just as well go back to the lobby, which would leave you waiting for... nothing.

**Comment by Phrancis** [comment]
Could we make the game server handle this and let the client only display it in response? Imagine a server-side logic like `while PlayerA is not ready OR PlayerB is not ready, push information message every X seconds" or something along those lines. Thoughts?

**Comment by jacwah** [comment]
I agree. The only way for a client to know their opponent is actually building a deck is having the server send "heart beats" every so often.

**Comment by danpantry** [comment]
Maybe we should send a notification to the server when the user opens/closes the deck building screen (however that works). The only thing that would be an issue is if the connection dies unexpectedly, but as this is a websocket connection we should be able to detect when the connection closes.

**Comment by jacwah** [comment]
>Alternatively, we could just require users to create their deck before a match (Hearthstone-style).

I think we should do this from a UX perspective. But that still requires waiting for your opponent to *choose* a deck (like in Hearthstone).

**Comment by Zomis** [comment]
I think that parts of what @danpantry said is already implemented. We need to investigate how it currently works (what messages are being sent between client and server) and see if we can use that to just show things better in the client. If there are any missing server messages, create issue in the main repository.
